qfdmo/models/acteur.py

```diff
@@ -542,6 +543,8 @@ class Meta:
         verbose_name = "ACTEUR de l'EC - AFFICHÉ"
         verbose_name_plural = "ACTEURS de l'EC - AFFICHÉ"
 
+    uuid = models.CharField(max_length=255, default=shortuuid.uuid, editable=False)
```

Comment:
```suggestion
    uuid = models.CharField(max_length=22, default=shortuuid.uuid, editable=False)
```

Par défaut ça fait 22 caractères de long les short uuid
https://pypi.org/project/shortuuid/

Reply:
j'ai hésité, donc soit !